server/tso/global_allocator.go

```diff
@@ -295,3 +260,13 @@ func (gta *GlobalTSOAllocator) getCurrentTSO() (pdpb.Timestamp, error) {
 func (gta *GlobalTSOAllocator) Reset() {
 	gta.timestampOracle.ResetTimestamp()
 }
+
+// GetDcLocations return all the dcLocations the GlobalTSOAllocator will check
+func (gta *GlobalTSOAllocator) GetDcLocations() []string {
+	dcLocationsMap := gta.allocatorManager.GetClusterDCLocations()
+	dcLocations := make([]string, len(dcLocationsMap))
```

Comment:
```suggestion
	dcLocations := make([]string, 0, len(dcLocationsMap))
```

Otherwise, the `dcLocations` will look like `["", "", "", "dc-1", "dc-2", "dc-1"]` eventually.

Reply:
updated.